.github/workflows/ci_cd.yml

```diff
@@ -93,6 +93,23 @@ jobs:
               exit 1
           fi
 
+  dependabot-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The token should have write permissions, otherwise it will fail

Reply:
Hm you're right. They changed this at some point I guess. Previously it was only read permissions. Thanks @klmcadams